setup.py

```diff
@@ -17,9 +17,14 @@ def readme():
         return f.read()
 
 
-INSTALL_REQUIRES = (
-    ['pandas', 'httplib2', 'google-api-python-client', 'oauth2client']
-)
+INSTALL_REQUIRES = [
+    'pandas',
+    'httplib2>=0.9.2',
+    'google-api-python-client>=1.6.0',
+    'google-auth>=1.0.0',
+    'google-auth-httplib2>=0.0.1',
```

Comment:
none of the google-auth* libraries are available on conda-forge. So the way forward is to form recipices for this (or better yet to get the google team to do this).